src/common-utils/main.sh

```diff
@@ -468,27 +478,31 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
                 "https://github.com/ohmyzsh/ohmyzsh" "${oh_my_install_dir}" 2>&1
 
             # Shrink git while still enabling updates
-            cd "${oh_my_install_dir}"
-            git repack -a -d -f --depth=1 --window=1
+            GIT_WORK_TREE="${oh_my_install_dir}" GIT_DIR="${oh_my_install_dir}/.git" git repack -a -d -f --depth=1 --window=1
         fi
 
         # Add Dev Containers theme
         mkdir -p ${oh_my_install_dir}/custom/themes
         cp -f "${FEATURE_DIR}/scripts/devcontainers.zsh-theme" "${oh_my_install_dir}/custom/themes/devcontainers.zsh-theme"
         ln -sf "${oh_my_install_dir}/custom/themes/devcontainers.zsh-theme" "${oh_my_install_dir}/custom/themes/codespaces.zsh-theme"
 
+        copy_to_user_files=()
         # Add devcontainer .zshrc template
         if [ "$INSTALL_OH_MY_ZSH_CONFIG" = "true" ]; then
             echo -e "$(cat "${template_path}")\nDISABLE_AUTO_UPDATE=true\nDISABLE_UPDATE_PROMPT=true" > ${user_rc_file}
             sed -i -e 's/ZSH_THEME=.*/ZSH_THEME="devcontainers"/g' ${user_rc_file}
+            copy_to_user_files+=("${oh_my_install_dir}")
+            [ -f "$user_rc_file" ] && copy_to_user_files+=("$user_rc_file")
+            chown -R ${USERNAME}:${group_name} "${copy_to_user_files[@]}"
+            OH_MY_ZSH_CONFIG_INSTALLED="true"
         fi
 
-        # Copy to non-root user if one is specified
-        if [ "${USERNAME}" != "root" ]; then
-            copy_to_user_files=("${oh_my_install_dir}")
-            [ -f "$user_rc_file" ] && copy_to_user_files+=("$user_rc_file")
+        # Copy to alternate user if one is specified
+        if [ "${USERNAME}" != "root" ] && (( ${#copy_to_user_files[@]} != 0 )); then
             cp -rf "${copy_to_user_files[@]}" /root
-            chown -R ${USERNAME}:${group_name} "${oh_my_install_dir}" "${user_rc_file}"
+            root_files=("/root/.oh-my-zsh")
+            [ -f /root/.zshrc ] && root_files+=("/root/.zshrc")
```

Comment:
Given root already has .zshrc installed ie previous step was `installOhMyZshConfig true`
When a new step is ran with `installOhMyZshConfig true`
Then don't copy .zshrc?